vespa-enforcer-extensions/src/test/java/com/yahoo/vespa/maven/plugin/enforcer/EnforceDependenciesAllProjectsTest.java

```diff
@@ -60,20 +62,20 @@ void fails_on_missing_dependency() {
                 Dependency.fromString("com.example:foo:1.2.3")));
         Path specFile = Paths.get("src/test/resources/allowed-dependencies.txt");
         var exception = assertThrows(EnforcerRuleException.class,
-                                     () -> validateDependencies(dependencies, specFile, "my-dep-enforcer"));
+                                     () -> validateDependencies(dependencies, specFile, POM_FILE, "my-dep-enforcer"));
         String expectedErrorMessage =
                 """
                 The dependency enforcer failed:
                 Removed dependencies:
                  - com.example:bar:2.3.4
-                Maven dependency validation failed. To update dependency spec execute following the command from root of aggregator pom:
-                $ mvn enforcer:enforce -DdependencyEnforcer.writeSpec -pl my-dep-enforcer
+                Maven dependency validation failed. To update dependency spec run:
```

Comment:
Suggest to change error message to something like:
"...validation failed. _If this change was intentional, update the dependency spec by running:_"